cypress/support/classes/board.js

```diff
@@ -0,0 +1,104 @@
+import board from "../elements/boardElements"
+import organization from "../elements/organizationElements"
+
+class Board {
+    createScrumBoard(name){
```

Comment:
To make this a bit more precise, I would name it something like `createScrumBoardAndAssertDetails()` or `createScrumBoardAndAssertName()` since you have an assertion 

`cy.get(board.boardList).should("be.visible").and("have.text", name); `

at the end.